yoyodyne/models/base.py

```diff
@@ -25,9 +25,9 @@ class BaseEncoderDecoder(pl.LightningModule):
     pad_idx: int
     start_idx: int
     # Sizes.
-    source_vocab_size: int
-    features_vocab_size: int
+    vocab_size: int
     target_vocab_size: int
+    features_vocab_size: int
```

Comment:
This order of the three is consistent across a bunch of things so you may want not to change it, even if you're changing the order in which they appear in the symbol table...